addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties

```diff
@@ -1,3 +1,6 @@
+exim.description = Import and Export functionality supporting multiple formats.
+exim.ui.name = Import/Export Add-on
```

Comment:
This is used in the extension though.

Reply:
I was trying to make it user friendly, since it is what’s shown in the GUI.

Is this one of those all add-one are extensions but not all extensions are add-ins kind of things?

(When I was looking at it last night I actually got wondering if we should be renaming that options panel. So that it’s more user friendly than strictly correct…)

Anyway I’m happy to switch it. Almost anything is better than the `NAME` constant(s).

Reply:
Yeah, the extensions are not add-ons. An add-on can have several extensions.